conduit/data/datamodules/vision/nico_plus_plus.py

```diff
@@ -0,0 +1,59 @@
+"""NICO++ data-module."""
+from typing import Any
+
+import albumentations as A  # type: ignore
+import attr
+from typing_extensions import override
+
+from conduit.data.datamodules.vision.base import CdtVisionDataModule
+from conduit.data.datasets.utils import stratified_split
+from conduit.data.datasets.vision import NICOPP, NicoPPTarget, SampleType
+from conduit.data.structures import TrainValTestSplit
+
+__all__ = ["NICOPPDataModule"]
+
+
+@attr.define(kw_only=True)
+class NICOPPDataModule(CdtVisionDataModule[NICOPP, SampleType]):
+    """Data-module for the NICO dataset."""
+
+    image_size: int = 224
+    superclass: NicoPPTarget = NicoPPTarget.AIRPLANE
+    make_biased: bool = True
+
+    @property
+    @override
+    def _default_train_transforms(self) -> A.Compose:
+        base_transforms = A.Compose(
+            [
+                A.Resize(self.image_size, self.image_size),
+                A.CenterCrop(self.image_size, self.image_size),
```

Comment:
I'm actually a bit confused about this (this is copied from NICO). Is the crop not superfluous?